### ISSUE TITLE
Montecarlo examples

### ISSUE THREAD

**Issue by ptheywood** [FLAMEGPU/FLAMEGPU#85]
- [x] Include the monte carlo examples 
- [x] add documentation (readme per example).
      added one for both
- [x] `.gitignore` should be updated to not include any generated output which does not want to be tracked (figures).

**Comment by mozhgan-kch** [comment]
- [ ] adding the report?
- [ ] updating the make files?

**Comment by ptheywood** [comment]
This can be closed once #115 has been resolved, and 1.5 is merged into master.
